paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsActivityTest.kt

```diff
@@ -263,16 +263,16 @@ internal class PaymentOptionsActivityTest {
         val args = PAYMENT_OPTIONS_CONTRACT_ARGS.updateState(isGooglePayReady = true)
         val viewModel = createViewModel(args)
 
-        val transitionTargets = mutableListOf<TransitionTarget>()
-        viewModel.transition.observeEventsForever { transitionTargets.add(it) }
+        val transitionTargets = mutableListOf<TransitionTarget?>()
+        viewModel.currentScreen.asLiveData().observeForever { transitionTargets.add(it) }
 
         activityScenario(viewModel).launch(createIntent(args)).use {
             it.onActivity {
                 idleLooper()
             }
         }
 
-        assertThat(transitionTargets).containsExactly(TransitionTarget.SelectSavedPaymentMethods)
+        assertThat(transitionTargets).containsExactly(null, TransitionTarget.SelectSavedPaymentMethods)
```

Comment:
If I remember right, you're making these `null` additions to signal an explicit loading screen. But planning to follow up with an initial screen of loading instead?

Reply:
Yep, I want to add a `TransitionTarget.Loading` to make the loading state more explicit. It’ll be the new initial value of the `StateFlow`.